boards/arm/stm32f746g_disco/stm32f746g_disco.dts

```diff
@@ -119,3 +120,67 @@
 		     &eth_txd0_pg13
 		     &eth_txd1_pg14>;
 };
+
+&flash0 {
```

Comment:
Is this related to qspi-nor or is it internal flash? It looks like something that might belong in a separate commit

Reply:
This part is related to internal flash. @erwango suggested I add it for consistency with other disco boards.

Reply:
I've removed the flash layout for internal flash and rebased.